daemons/front_stats_update_daemon.rb

```diff
@@ -2,15 +2,38 @@
 
 require_relative "../config/environment"
 
+networks = %w[mainnet testnet]
+sleep_time = 5 # seconds
+update_stats_time = 60 # seconds
+@counter = 0
+
+def get_latest_cluster_stats
+  {
+    mainnet: ClusterStat.by_network("mainnet").last&.attributes,
+    testnet: ClusterStat.by_network("testnet").last&.attributes
+  }
+end
+
+@last_cluster_stats = get_latest_cluster_stats
+
 loop do
   begin
+    @counter += 1
+    if @counter == update_stats_time / sleep_time
```

Comment:
I'm not getting the point. 
The variables update_stats_time and sleep_time are constant and will not change. So executing the above `update_stats_time / sleep_time` will return value of 12.

Can we just define `update_stats_time` only and compare it with the `@counter`?

Reply:
if sleep time changes we still want to have 1 minute between updates